packages/types/types/index.d.ts

```diff
@@ -147,6 +154,38 @@ export interface ThemeProviderProps {
   customBreakpoints?: string[]
 }
 
+export interface ColorValidation {
+  color?: PropValidator
```

Comment:
Would `TextColorProps` from `styled-system` work instead?

Reply:
Does `TextColorProps` handle what we're doing to validate if a color is semantic or not?

Reply:
No, I've just been using `TextColorProps` until we can replace it everywhere with whatever custom color types we come up with